src/main/docker/Dockerfile

```diff
@@ -15,8 +15,10 @@
 #
 
 ARG DOCKER_HUB_PUBLIC=docker.io
-FROM ${DOCKER_HUB_PUBLIC}/digitallyseamless/nodejs-bower-grunt:5
+FROM ${DOCKER_HUB_PUBLIC}/cafapi/opensuse-nodejs10:2
 
+RUN zypper install -y gzip && \
+    zypper install -y -t pattern devel_basis
```

Comment:
https://github.com/CAFapi/doc-site-compiler-image/pull/5#discussion_r729055950

If you're now installing `tar` using `zypper` (implicitly via `devel_basis`), then do we still need to download and compile it just below?